stores/sql/database.go

```diff
@@ -0,0 +1,20 @@
+package sql
+
+import (
+	"context"
+	"io"
+)
+
+// The database interfaces define all methods that a SQL database must implement
+// to be used by the SQLStore.
+type (
+	Database interface {
+		io.Closer
+
+		Version(ctx context.Context) (string, string, error)
+	}
+
+	MetricsDatabase interface {
```

Comment:
I don't get why `MetricsDatabase` is a different interface. Especially since I think in consecutive PRs we update this to match.